projects/packages/publicize/src/class-publicize.php

```diff
@@ -250,30 +250,12 @@ public function get_all_connections_for_user( $args = array() ) {
 					$user_id = (int) $connection['connection_data']['user_id'];
 					// phpcs:ignore WordPress.PHP.YodaConditions.NotYoda
 					if ( $user_id === 0 || $this->user_id() === $user_id ) {
-						if ( $this->use_admin_ui_v1() ) {
-							$connections_to_return[] = array_merge(
-								$connection,
-								array(
-									'service_name'   => $service_name,
-									'connection_id'  => $connection['connection_data']['id'],
-									'can_disconnect' => self::can_manage_connection( $connection['connection_data'] ),
-									'profile_link'   => (string) $this->get_profile_link( $service_name, $connection ),
-									'shared'         => '0' === $connection['connection_data']['user_id'],
-									'status'         => 'ok',
-								)
-							);
-						} else {
-							$connections_to_return[ $service_name ][ $id ] = $connection;
-						}
+						$connections_to_return[ $service_name ][ $id ] = $connection;
```

Comment:
I might be confused here, but use_admin_ui_v1 was true for all sites now because they have connection management right? fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Swrgcnpx%2Qcyhtva%2Sfha%2Swrgcnpx_iraqbe%2Snhgbznggvp%2Swrgcnpx%2Qchoyvpvmr%2Sfep%2Spynff%2Qchoyvpvmr%2Qonfr.cuc%3Se%3Q54q2o8q6%23285-og

If so, shouldn't we keep the other part of the if?

Reply:
Yes, that is true for all sites now and that method is there for backwards compatibility for older version mismatch.

We can remove this method after some time.